src/Fixer/Alias/NoMixedEchoPrintFixer.php

```diff
@@ -80,7 +80,7 @@ public function getDefinition()
      */
     public function getPriority()
     {
-        return -10;
+        return 0;
```

Comment:
1. the changes to existing priorities, especially **to** or **from** neutral (`0`), may be really problematic for all custom rules that had priority set up in reference to number from built-in fixer.
2. when working on any new approach how to build priority system that is not based on hardcoded numbers, we need to think how custom fixers will integrate with new priority system in stable manner (eg, we cannot regenerate the numbers every time we add a new fixer with priority relation, as it will doom all custom fixers)